### ISSUE TITLE
Term Serialization between Elixir and JavaScript

### ISSUE THREAD

**Issue by bryanjos** [elixirscript/elixirscript#373]
Not sure if [BERT](http://bert-rpc.org/) is the best route or not, but it is an option. The js implementation I saw didn't support maps though. We would need something that deserialized on the JavaScript side to the classes we have for Tuples, Bitstrings, etc

**Comment by OvermindDL1** [comment]
Adding map support would not be hard.  The Erlang Term Format is very well documented and easy to parse.

**Comment by OvermindDL1** [comment]
Could just use the ETF straight.

**Comment by bryanjos** [comment]
I agree ETF is probably best. Is this the documentation? http://erlang.org/doc/apps/erts/erl_ext_dist.html

**Comment by OvermindDL1** [comment]
That is it precisely.  :-)

There are a couple javascript libraries around that parse it too of various completion.
